### PR TITLE
Updated with support for an ini file (specified on the command line)

### PR DESCRIPTION
**Pull request by weberr13**
That will override key/value if needed

### CODE REVIEW

File: c/c.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+// readFromBuffer given and populate the conf
+func readFromBuffer(data []byte) (c Conf) {
+	c.ConfigOverrides = make(map[string]string)
+	entries := strings.Split(string(data), "\n")
+	for _, line := range entries {
+		keyAndValue := strings.Split(line, "=")
+		if len(keyAndValue) > 1 {
+			var buffer bytes.Buffer
+			buffer.WriteString(keyAndValue[1])
+			for i := 2; i < len(keyAndValue); i++ {
```
Comment (vrecan) [inline c/c.go:43]: this doesn't look like it will handle = signs in the value

**Comment by tartale** [comment]
Can we see an example `c.ini` file?

File: c/c.go (hunk 1 above)
Comment (vrecan) [inline c/c.go:43]: ignore me looks like it works

**Comment by vrecan** [comment]
:+1:

File: merge/mergeRaw.go [hunk 2]
```diff
@@ -120,7 +122,21 @@ func Combine(src *Merge, dst *Merge) *Merge {
...
+	for k, v := range conf.ConfigOverrides {
+		if strings.Contains(d.Key, k) {
+			var buffer bytes.Buffer
+			buffer.WriteString(string([]rune(d.Value)[0]))
+			buffer.WriteString(v)
+			d.Value = buffer.String()
+			return
+		}
+	}
+}
```
Comment (tartale) [inline merge/mergeRaw.go:142]: Would it be really difficult to tie the file name that you're merging into the MergeForward config?

Not seeing a config file; I'm assuming that this is a straight "key match", which might bite us if we mean to over-write a specific field in one config file that happens to have the same name in another.

We have so many fields, and so many developers, that this is a bug that is probably going to creep up.

Having the file name be part of it narrows the scope

**Comment by tartale** [comment]
:+1: looks good, but consider my suggestion (also I'd like to see what the ini file looks like?)

File: merge/mergeRaw.go (hunk 2 above)
Comment (vrecan) [inline merge/mergeRaw.go:142]: to handle each config you want to merge and overide. you can provide it a different ini file for that merge.

**Pull request merged**